src/components/list/list.js

```diff
@@ -129,6 +129,22 @@ function mdListItemDirective($mdAria, $mdConstant, $mdUtil, $timeout) {
         }
       }
 
+      function setupProxiedMenu() {
+        var menuEl = angular.element(proxyElement);
+
+        if (proxyElement.parentNode.firstElementChild == proxyElement) {
+          menuEl.attr('md-position-mode', 'target target');
+        } else {
+          menuEl.attr('md-position-mode', 'target-right target');
+        }
+
+        var buttonEl = menuEl.find('md-button') || menuEl.find('button');
+        if (!buttonEl || !buttonEl[0].parentNode == menuEl[0]) return;
+
+        if (!buttonEl[0].hasAttribute('ng-click')) buttonEl[0].setAttribute('ng-click', '$mdOpenMenu($event)');
```

Comment:
Using `.attr` isn't working for `ng-click`, seems strange. 

The code is old, I will change a few things.

The initial thought was, adding support for show a `md-menu` as a `second element` / `second action` / proxy element.